lib/contentful_model/migrations/content_type.rb

```diff
@@ -6,10 +6,15 @@ def initialize(name = nil, management_content_type = nil)
         @management_content_type = management_content_type
       end
 
+      def id(id)
+        @id = id
+      end
+
       def save
         if new?
           @management_content_type = management.content_types.create(
             ContentfulModel.configuration.space,
+            id: @id || camel_case(@name),
```

Comment:
Just a heads up, I did something similar in .NET earlier and someone actually wanted to have the randomized id... don't remember the exact reasoning. Just mentioning it.

Reply:
Thanks, I had a similar request before for the database importer to be able to generate Base62 random strings as well, but for the moment I'll leave it like this, and in case someone requests not to have it, I'll add an override to this.

I personally think it's a good addition to be able to set human readable IDs by default.

Reply:
Agreed!

Reply:
@dlitvakb the only thing is this creates IDs differently to the web interface, e.g., for `Some Content Type` it generates `SomeContentType`, whereas the web client generates `someContentType`.

Reply:
Oh, you're right, if `name` starts with a capital, it will respect it.

I'll fix that and make it lower case always.

Cheers